en/SUMMARY.md

```diff
@@ -25,7 +26,8 @@
   * [Mock Link](tools/mock_link.md)
 * [Command Line Options](command_line_options.md)
 * [Custom Builds](custom_build/custom_build.md)
-  * [Updating custom builds from QGC repro](custom_build/upstream_merge.md) 
+  * [Git Branching for Custom Builds](custom_build/GitBranching.md)
```

Comment:
Yes, I know "For custom builds" is implied. But it is an annoying factor of gitbook search that the context is only the text when you search".